src/libstore/local-store.cc

```diff
@@ -1306,11 +1306,11 @@ void LocalStore::addToStore(const ValidPathInfo & info, Source & source,
             auto hashResult = hashSink.finish();
 
             if (hashResult.first != info.narHash)
-                throw Error("hash mismatch importing path '%s';\n  specified: %s\n  got:       %s",
+                throw Error("hash mismatch importing path '%s' (try `nix store repair`);\n  specified: %s\n  got:       %s",
```

Comment:
```suggestion
                throw Error("hash mismatch importing path '%s' (try 'nix store repair');\n  specified: %s\n  got:       %s",
```

Error messages are not markdown.

Reply:
The other day I ran into an error message that along the lines of

    'something is wrong, did you mean mapAttrs'

That reads a lot like `mapAttrs'`, a valid identifier and existing function. Backticks would resolve that confusion.
I also think that people read unrendered markdown quite often, when browsing markdown-documented code for example. Maybe the backtick is becoming a natural way to quote code.

That's a larger decision that shouldn't be made here though, so I do agree with the suggestion.